index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Salmon Cookies</title>
+</head>
+<body>
+  <header>
+    <h1>Salmon Cookies</h1>
+    <img src='salmon.png'/>
+  </header>
+  <nav>
+    <a href='sales.html'>sales</a>
```

Comment:
Also, consider adding additional nav links to cover all pages in your app on all pages (e.g. home and sales on both index.html and sales.html)